### PR TITLE
fix absl import issue #25282

### PR DESCRIPTION
**Pull request by zjjott**
fix issue when using:

```
from tensorflow.python.platform.googletest import mock
```
there is code in `tensorflow/python/platform/googletest.py`

```
from absl.testing.absltest import *
```
but absl.testing.absltest have `mock` object after absl>=0.4.0

this case is occur after commit 87cc788e1cc
@revan 
87cc788e1cc (Revan Sopher           2019-01-23 13:14:20 -0800

https://github.com/tensorflow/tensorflow/issues/25282

### CODE REVIEW

**Comment by revan** [comment]
Thanks for the fix! I think we should be able to bump this all the way to 0.7.0, after this commit: https://github.com/tensorflow/tensorflow/commit/661103ed87cdf0f5a792100b543006b71d62697a

Can you try 0.7.0?

**Comment by zjjott** [comment]
@revan OK,I have rebased this commit to `absl-py==0.7.0`

**Comment by zjjott** [comment]
@revan Hello,is this PR in progress

**Comment by revan** [comment]
Hi @zjjott, I've retriggered the tests and am waiting for a green signal.

**Review by revan — APPROVED** [review]

**Comment by revan** [comment]
@rthadur submit failed due to unrelated linter error, can we force?

**Comment by rthadur** [comment]
> @rthadur submit failed due to unrelated linter error, can we force?

@revan we will wait until it gets merged internally.

**Pull request merged**
